src/debian/snmp_tempdisk.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This file is never copied on the system.
A line must be added in the playbook to copy it in the hypervisor.

Reply:
the file is copied by this task/line : https://github.com/seapath/ansible/blob/2311d8fcbafc81fb40c7fbbd34c9712d179dd919/roles/debian/tasks/main.yml#L228C13-L228C13